packages/account-manifest/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "account-manifest",
```

Comment:
suggestion: It feels like there's a more descriptive name for this, something like `frame-manifest-generator` or `miniapp-manifest-generator`. More verbose but we can always alias from the root.